.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,55 @@
+name: "Build and deploy farm stack on azure"
```

Comment:
Unsure if you want separate workflows for building and deploying, maybe trigger the latter on the creation of a release / tag?